locksmith/src/routes/v2/images.ts

```diff
@@ -80,4 +106,15 @@ router.post(
   }
 )
 
+router.get('/image/:name', (req, res) => {
```

Comment:
Please make sure this route is set *only* on dev. Also, let's do `/file/:name` because this can eventually be used for things that are not images.